resources/keycloak/custom-user-provider/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM quay.io/keycloak/keycloak:25.0.6 as builder
+
+# Enable health and metrics support
+ENV KC_HEALTH_ENABLED=true
+ENV KC_METRICS_ENABLED=true
+
+# Configure a database vendor
+ENV KC_DB=postgres
+
+WORKDIR /opt/keycloak
+
+COPY ./target/keycloak-custom-providers-0.1.0-SNAPSHOT.jar /opt/keycloak/providers/
+
+# for demonstration purposes only, please make sure to use proper certificates in production instead
+RUN keytool -genkeypair -storepass password -storetype PKCS12 -keyalg RSA -keysize 2048 -dname "CN=server" -alias server -ext "SAN:c=DNS:localhost,IP:127.0.0.1" -keystore conf/server.keystore
+RUN /opt/keycloak/bin/kc.sh build
+
+FROM quay.io/keycloak/keycloak:25.0.6
+COPY --from=builder /opt/keycloak/ /opt/keycloak/
+
+# change these values to point to a running postgres instance
+ENV KC_DB=postgres
+ENV KC_DB_URL=jdbc:postgresql://192.168.110.147/postgres?currentSchema=keycloak
+ENV KC_DB_USERNAME=postgres
+ENV KC_DB_PASSWORD=postgres
+ENV KC_HOSTNAME=localhost
+ENV KC_HOSTNAME=localhost
+ENV KC_LOG_LEVEL=DEBUG
```

Comment:
This was only used for local dev, and made more sense when it was a separate project entirely. Now that it is in the cvmanager, it will only be run through the parent keycloak image. I chose to remove it entirely